docs/platforms/apple/common/features/experimental-features.mdx

```diff
@@ -4,45 +4,22 @@ sidebar_order: 2
 description: "Learn about the experimental features available for Sentry's Apple SDK."
 ---
 
+<Alert>
+  Experimental features are still a work-in-progress and may have bugs. We
+  recognize the irony.
+</Alert>
+
 Do you want to try some new experimental features? On the latest version of the Apple SDK, you can:
 
-- Enable <PlatformLink to="/tracing/instrumentation/automatic-instrumentation/#time-to-full-display">Time to Full Display (TTFD)</PlatformLink> to gain insight into how long it takes your view controller to launch and load all of its content.
 <PlatformSection notSupported={["apple.tvos", "apple.watchos", "apple.visionos"]}>
 - Enable <PlatformLink to="/profiling/#enable-launch-profiling">App Launch Profiling</PlatformLink> to get detailed profiles for your app launches.
 - Enable <PlatformLink to="/profiling/#continuous-profiling">Continuous Profiling</PlatformLink> to get full coverage of your app's execution.
 </PlatformSection>
+<PlatformSection notSupported={["apple.macos", "apple.watchos", "apple.visionos"]}>
+- Enable <PlatformLink to="/configuration/app-hangs/#app-hangs-v2">AppHangsV2</PlatformLink> to get more detailed app hangs. This differentiates between fully-blocking and non-fully-blocking app hangs, and also tells you the duration of app hangs.
+</PlatformSection>
 - If you use Swift concurrency, stitch together stack traces of your async code with the `swiftAsyncStacktraces` option. Note that you can enable this in your Objective-C project, but only async code written in Swift will be stitched together.
-- Enable <PlatformLink to="/configuration/app-hangs/#app-hangs-v2">AppHangsV2</PlatformLink> to get more detailed app hangs. The main difference is that AppHangsV2 differentiates between fully-blocking and non-fully-blocking app hangs and measures the duration of app hangs.
-
-<Alert>
-  Experimental features are still a work-in-progress and may have bugs. We
-  recognize the irony.
-</Alert>
-
-```swift {tabTitle:Swift}
-import Sentry
-
-SentrySDK.start { options in
-    // ...
-
-    // Enable all experimental features
-    options.enableTimeToFullDisplayTracing = true
-    options.enableAppLaunchProfiling = true
-    options.swiftAsyncStacktraces = true
-}
-```
-
-```objc {tabTitle:Objective-C}
```

Comment:
removed the snippet because it's annoying to maintain and most features only apply to specific platforms, which would mean maintaining multiple snippets to stay correct. also most features require some context and setup so ppl should look at the feature docs pages